moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -326,7 +346,15 @@ const ServoParameters& Servo::getParameters() const
 
 sensor_msgs::JointStateConstPtr Servo::getLatestJointState() const
 {
-  return joint_state_subscriber_->getLatest();
+  auto msg = moveit::util::make_shared_from_pool<sensor_msgs::JointState>();
+  auto current_state_time = planning_scene_monitor_->getStateMonitor()->getCurrentStateAndTime();
+
+  msg->header.stamp = current_state_time.second;
+  msg->name = current_state_time.first->getJointModelGroup(parameters_.move_group_name)->getActiveJointModelNames();
+  current_state_time.first->copyJointGroupPositions(parameters_.move_group_name, msg->position);
+  current_state_time.first->copyJointGroupPositions(parameters_.move_group_name, msg->velocity);
```

Comment:
Should be copyJointGroupVelocities, right?

Reply:
good catch